testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -613,6 +613,24 @@ class PlatformViewForTouchIOSScenario extends Scenario
   }
 }
 
+/// A simple platform view for testing platform view with a continuous texture layer.
```

Comment:
what does continuous texture means? Does it mean a texture that simulates a video being played?